### PR TITLE
Upgrade Django version to 4.2 and fix a flake8 issue.

### PR DESCRIPTION
**Pull request by somexpert**
ping @ldko, @gracieflores, this is ready for review.

### CODE REVIEW

File: tests/factories.py [hunk 1]
```diff
@@ -71,7 +71,7 @@ def subjects(self, create, extracted, **kwargs):
                 self.subjects.add(subject)
 
     @factory.post_generation
-    def coverage(self, create, extracted, **kwargs):
+    def coverage_locs(self, create, extracted, **kwargs):
```
Comment (ldko) [inline tests/factories.py:74]: Is this the flake8 issue you mentioned? What was the issue?

Comment (somexpert) [reply tests/factories.py:74]: `coverage` the attribute (line 60) being overridden by `coverage` the method was the issue

**Review by ldko — APPROVED** [review]
Looks good to me. Tests passed, ran locally.

**Review by gracieflores — APPROVED** [review]
Looks good. 👍🏼

**Pull request merged**